docs/sdks/flutter/GETTING_STARTED.md

```diff
@@ -0,0 +1,115 @@
+## Getting Started
+
+### Add your Flutter Platform
+To init your SDK and start interacting with Appwrite services, you need to add a new Flutter platform to your project. To add a new platform, go to your Appwrite console, choose the project you created in the step before, and click the 'Add Platform' button.
+
+From the options, choose to add a new **Flutter** platform and add your app credentials. Appwrite Flutter SDK currently supports building apps for both iOS and Android.
+
+If you are building your Flutter application for multiple devices, you have to follow this process for each different device.
+
+#### iOS
+For **iOS** add your app name and Bundle ID, You can find your Bundle Identifier in the General tab for your app's primary target in Xcode.
+
+### Android
```

Comment:
I got confused in the header hierarchy here: Why are the Android headers `h3` and iOS/Web `h4`?